app/views/distributions/leap.html.erb

```diff
@@ -1,4 +1,7 @@
 <% version = '42.3' %>
+<% @page_title = _('openSUSE Leap') + " " + version -%>
```

Comment:
you can also write it into one string, in my opinion that is a bit cleaner:
`<% @page_title = "#{_('openSUSE Leap')} #{version}" %>`